ProcessLib/LIE/Common/FractureProperty.h

```diff
@@ -54,11 +59,28 @@ inline void setFractureProperty(unsigned dim, MeshLib::Element const& e,
     // 1st node is used but using other node is also possible, because
     // a fracture is not curving
     for (int j = 0; j < 3; j++)
-        frac_prop.point_on_fracture[j] = e.getNode(0)->getCoords()[j];
+        frac_prop.point_on_fracture[j] = e.getCenterOfGravity().getCoords()[j];
     computeNormalVector(e, dim, frac_prop.normal_vector);
     frac_prop.R.resize(dim, dim);
     computeRotationMatrix(e, frac_prop.normal_vector, dim, frac_prop.R);
 }
 
+
+inline void setBranchProperty(
```

Comment:
can be converted to a constructor of BranchProperty.

Reply:
here i intentionally separate data and functions (to let myself focus on the data structure during coding).

Reply:
I see.